app/src/info/CooperativeAddOnItemRequirementsGenerator.ts

```diff
@@ -56,49 +78,17 @@ export default class AddOnItemRequirementsGenerator implements IProjectInfoItemG
           await rpManifest.load();
 
           if (!rpManifest.packScope) {
+            // CADDONIREQ170
             items.push(
               new ProjectInfoItem(
                 InfoItemType.testCompleteFail,
                 this.id,
-                140,
+                CooperativeAddOnItemRequirementsGeneratorTest.ResourcePackDoesNotHavePackScopeWorld,
```

Comment:
170 now instead of 140. Intentional?

Reply:
This might have been another case where I had duplicate identifers for two different errors that is now straightened out :)